pkg/istio/rbac/common/servicerolebinding.go

```diff
@@ -99,9 +99,18 @@ func GetServiceRoleBindingSpec(k8sRoleName string, members []*zms.RoleMember, en
 	}
 
 	if len(subjects) == 0 {
-		return nil, fmt.Errorf("no subjects found for the ServiceRoleBinding: %s", k8sRoleName)
+		log.Warningln("no subjects found for the ServiceRoleBinding: %s", k8sRoleName)
 	}
 
+	//add role spiffee for role certificate
+	if athenzDomainName == "" || roleName == "" {
+		return nil, fmt.Errorf("empty string found in athenzDomainName: %s and roleName: %s", athenzDomainName, roleName)
+	}
+	spiffeSubject := &v1alpha1.Subject{
+		User: fmt.Sprintf("%s/ra/%s", athenzDomainName, roleName),
```

Comment:
could you move this to common/util.go and have a function like `RoleToSpiffe` similar to https://github.com/yahoo/k8s-athenz-istio-auth/blob/master/pkg/istio/rbac/common/util.go#L29 ?